server/backend/api/app/routers/router_bcsc_proxy.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from fastapi import APIRouter, Request
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+import requests
+from jose import jwt
+from fastapi import HTTPException
+import json
+from .. import kms_lookup
+from .. import config
+
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("/dev/userinfo", status_code=200)
+def bcsc_userinfo_dev(request: Request):
+    return bcsc_userinfo(request, "https://idtest.gov.bc.ca/oauth2/userinfo")
+
+
+@router.get("/test/userinfo", status_code=200)
+def bcsc_userinfo_test(request: Request):
+    return bcsc_userinfo(request, "https://idtest.gov.bc.ca/oauth2/userinfo")
+
+
+@router.get("/prod/userinfo", status_code=200)
+def bcsc_userinfo_prod(request: Request):
```

Comment:
Proxy endpoints are not secure endpoints, is it intended? (maybe that's why calling to "/userinfo" returns encrypted data?)